README.md

```diff
@@ -67,7 +67,7 @@ if you need a production data please contact our [sales team](mailto:rpo@adorsys
     - **<path/to/keystore.p12>** with your keystore file location
     - **<path/to/adapter.config.properties>** with your adapter config file
     - **<path/to/aspsp-adapter-config.csv>** with your aspsp-registry configuration file
-    - **<keystore-password>** with a keystore password
+    - __<keystore-password>__ with a keystore password
```

Comment:
unfortunately, this fix also doesn't work https://github.com/adorsys/xs2a-adapter/blob/93dad027f9959331aa7cdb4b8313daa854dd8b8b/README.md

Reply:
what a glitch!
looking into it

Reply:
fixed